src/main/java/de/zalando/aruha/nakadi/repository/db/SubscriptionDbRepository.java

```diff
@@ -0,0 +1,82 @@
+package de.zalando.aruha.nakadi.repository.db;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import de.zalando.aruha.nakadi.domain.Subscription;
+import de.zalando.aruha.nakadi.exceptions.DuplicatedSubscriptionException;
+import de.zalando.aruha.nakadi.exceptions.InternalNakadiException;
+import de.zalando.aruha.nakadi.exceptions.NoSuchSubscriptionException;
+import org.springframework.dao.DuplicateKeyException;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+
+import java.io.IOException;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Set;
+
+import static com.google.common.collect.Sets.newTreeSet;
+
+public class SubscriptionDbRepository extends AbstractDbRepository {
+
+    private final SubscriptionMapper rowMapper = new SubscriptionMapper();
+
+    public SubscriptionDbRepository(final JdbcTemplate jdbcTemplate, final ObjectMapper objectMapper) {
+        super(jdbcTemplate, objectMapper);
+    }
+
+    public void saveSubscription(final Subscription subscription) throws InternalNakadiException,
```

Comment:
What is the plan for usage this method?
From my point of view it must be like that:
```
id = saveSubscription(subscriptionData)
```